pkg/network/tracer/tracer_linux_test.go

```diff
@@ -2068,3 +2069,67 @@ func TestBlockingReadCounts(t *testing.T) {
 
 	assert.Equal(t, uint64(n), conn.MonotonicSum().RecvBytes)
 }
+
+// GoTLS test
+
+func TestHTTPGoTLSCaptureNewProcess(t *testing.T) {
+	const ClientBin string = "./testutil/cmd/gotls_client/gotls_client"
+	const ServerAddr string = "localhost:8081"
+	const ExpectedOccurrences int = 10
+
+	if !httpSupported(t) {
+		t.Skip("HTTPS feature not available on pre 4.14.0 kernels")
+	}
+
+	if !httpsSupported(t) {
+		t.Skip("HTTPS feature not available supported for this setup")
+	}
+
+	// Given
+	var closeServer func() = httptest.HTTPServer(t, ServerAddr, httptest.Options{
+		EnableTLS: true,
+	})
+
+	cfg := config.New()
+	cfg.EnableHTTPMonitoring = true
+	cfg.EnableHTTPSMonitoring = true
+	cfg.BPFDebug = true
+
+	tr, err := NewTracer(cfg)
+	require.NoError(t, err)
+	defer tr.Stop()
+	require.NoError(t, tr.RegisterClient("1"))
+
+	// When
+	req, err := nethttp.NewRequest(nethttp.MethodGet, fmt.Sprintf("https://%s/%d/request", ServerAddr, nethttp.StatusOK), nil)
+	require.NoError(t, err)
+
+	clientBuildCmd := fmt.Sprintf("go build -o %s ./testutil/cmd/gotls_client", ClientBin)
```

Comment:
use `CurDir`

Reply:
Done. Required fixing `CurDir` so that it actually returns the caller's current directory (it returned the directory of the file containing `CurDir`'s implementation before).